src/store/Configuration/mutations.ts

```diff
@@ -0,0 +1,76 @@
+import { ConfigurationStateIF } from '@/interfaces'
+import axios from '@/axios-auth'
+
+export default {
+  setConfiguration (state: ConfigurationStateIF, data: any) {
+    state.configuration = data
+  },
+  setSessionVariables (state: ConfigurationStateIF, responseData: any) {
+    const authWebUrl: string = responseData['AUTH_WEB_URL']
+    sessionStorage.setItem('AUTH_WEB_URL', authWebUrl)
+    console.info('Set Auth Web URL to: ' + authWebUrl)
+
+    const registryHomeUrl: string = responseData['REGISTRY_HOME_URL']
+    sessionStorage.setItem('REGISTRY_HOME_URL', registryHomeUrl)
+    console.info('Set Registry Home URL to: ' + registryHomeUrl)
+
+    const businessesUrl: string = responseData['BUSINESSES_URL']
+    sessionStorage.setItem('BUSINESSES_URL', businessesUrl)
+    console.info('Set Businesses URL to: ' + businessesUrl)
+
+    const createUrl: string = responseData['BUSINESS_CREATE_URL']
+    sessionStorage.setItem('CREATE_URL', createUrl)
+    console.info('Set Create URL to: ' + createUrl)
+
+    const editUrl: string = responseData['BUSINESS_EDIT_URL']
+    sessionStorage.setItem('EDIT_URL', editUrl)
+    console.info('Set Edit URL to: ' + editUrl)
+
+    const authApiUrl: string = responseData['AUTH_API_URL'] + responseData['AUTH_API_VERSION'] + '/'
+    sessionStorage.setItem('AUTH_API_URL', authApiUrl)
+    console.info('Set Auth API URL to: ' + authApiUrl)
+
+    const payApiUrl: string = responseData['PAY_API_URL'] + responseData['PAY_API_VERSION'] + '/'
+    sessionStorage.setItem('PAY_API_URL', payApiUrl)
+    console.info('Set Pay API URL to: ' + payApiUrl)
+
+    // for system alert banner (sbc-common-components)
+    const statusApiUrl: string = responseData['STATUS_API_URL'] + responseData['STATUS_API_VERSION']
+    sessionStorage.setItem('STATUS_API_URL', statusApiUrl)
+    console.info('Set Status API URL to: ' + statusApiUrl)
+
+    const keycloakConfigPath: string = responseData['KEYCLOAK_CONFIG_PATH']
+    sessionStorage.setItem('KEYCLOAK_CONFIG_PATH', keycloakConfigPath)
+    console.info('Set KeyCloak Config Path to: ' + keycloakConfigPath)
+
+    const siteminderLogoutUrl: string = responseData['SITEMINDER_LOGOUT_URL']
+    if (siteminderLogoutUrl) {
+      sessionStorage.setItem('SITEMINDER_LOGOUT_URL', siteminderLogoutUrl)
+      console.info('Set Siteminder Logout Url to: ' + siteminderLogoutUrl)
```

Comment:
All these `console.info()` were helpful in earlier days, but maybe not needed now?

<img width="359" alt="image" src="https://user-images.githubusercontent.com/10002889/218569680-882e33f8-fb54-42c0-9655-9a6d0a89a212.png">

Personally, I like the "initializing..." and "starting..." messages (and "starting app should/could tell us which app this is), but I think the other messages are no longer needed. Thoughts?